### PR TITLE
ruff: ARG, no unused arguments

### PR DESCRIPTION
**Pull request by ahms5**
see #647

### CODE REVIEW

File: pyfar/samplings/external/eq_area_partitions.py
Comment (f-brinkmann) [inline pyfar/samplings/external/eq_area_partitions.py:1]: see comment above - do we want to explude this file from our checks?

**Review by f-brinkmann — CHANGES_REQUESTED** [review]
Thanks for the effort. I commented on two files, that we could think about excluding. I'm also wondering about the class-methods where we don't use `self` and you added `# noqa: ARG002`. Would it be more elegant to use the `@staticmethod` decorator and remove `self` completely from those methods?

File: pyfar/samplings/external/eq_area_partitions.py [hunk 1]
```diff
@@ -108,6 +108,7 @@ def point_set_polar(dimension, N):
             a_point = (a_top + a_bot)/2
 
             point_l_n = np.arange(0, np.size(points_l), dtype=int)
+            # points_l = points_l[np.newaxis]
```
Comment (ahms5) [inline pyfar/samplings/external/eq_area_partitions.py:111]:
```suggestion
```

File: pyfar/samplings/external/eq_area_partitions.py [hunk 2]
```diff
@@ -120,6 +121,7 @@ def point_set_polar(dimension, N):
                     points_l[:, point_l_n]
 
             points_s[dimension-1, point_n+point_l_n-1] = a_point
+            # point_n = point_n + points_l.shape[1]
```
Comment (ahms5) [inline pyfar/samplings/external/eq_area_partitions.py:124]:
```suggestion
```

File: pyfar/samplings/external/eq_area_partitions.py [hunk 3]
```diff
@@ -201,6 +203,8 @@ def polar_colat(dimension, N):
     colatitude : double
         The colatitude angle of the top cap.
     """
+    # enough = N > 2
+    # c_polar = np.empty()
```
Comment (ahms5) [inline pyfar/samplings/external/eq_area_partitions.py:207]:
```suggestion
```

File: pyfar/dsp/filter/_audiofilter.py [hunk 4]
```diff
@@ -871,6 +871,7 @@ def zplane_plot(ax, z, p, k):
     plt.text(0, +1, 'k=%f' % k)
     plt.text(0, -1, 'ROC for causal: white')
     plt.axis('square')
+    # plt.axis([-2, 2, -2, 2])
```
Comment (ahms5) [inline pyfar/dsp/filter/_audiofilter.py:874]:
```suggestion
```

**Review by f-brinkmann — DISMISSED** [review]

**Pull request merged**